gsa/src/gmp/models/__tests__/certbund.js

```diff
@@ -0,0 +1,278 @@
+/* Greenbone Security Assistant
+ *
+ * Authors:
+ * Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+ *
+ * Copyright:
+ * Copyright (C) 2018 Greenbone Networks GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+/* eslint-disable max-len */
+
+import CertBundAdv from 'gmp/models/certbund';
+import Info from 'gmp/models/info';
+import {testModel} from 'gmp/models/testing';
+
+import {parseDate} from 'gmp/parser';
+
+testModel(CertBundAdv, 'certbund');
+
+describe('CertBundAdv model tests', () => {
+
+  test('should be instance of Info', () => {
+    const certBundAdv = new CertBundAdv({});
+
+    expect(certBundAdv).toBeInstanceOf(Info);
+  });
+
+  test('should parse severity', () => {
+    const elem = {
+      max_cvss: '8.5',
+    };
+    const certBundAdv = new CertBundAdv(elem);
+
+    expect(certBundAdv.severity).toEqual(8.5);
+    expect(certBundAdv.max_cvss).toBeUndefined();
+  });
+
+  test('should return empty categories array if no advisory is given', () => {
+    const certBundAdv = new CertBundAdv({});
+    expect(certBundAdv.categories).toEqual([]);
+  });
+
+  test('should return array of categories', () => {
+    const elem = {
+      raw_data: {
+        Advisory: {
+          CategoryTree: ['foo', 'bar'],
+        },
+      },
+    };
+    const certBundAdv = new CertBundAdv(elem);
+    expect(certBundAdv.categories).toEqual(['foo', 'bar']);
+  });
+
+  test('should return string if single CategoryTree is given', () => {
+    const elem = {
+      raw_data: {
+        Advisory: {
+          CategoryTree: 'foo',
+        },
+      },
+    };
+    const certBundAdv = new CertBundAdv(elem);
+    expect(certBundAdv.categories).toEqual('foo');
```

Comment:
I guess you have spotted bug in CertBundAdv model. This should be
```js
['foo']
```
Could you fix the model in an additional commit/pr?